### PR TITLE
(ru) Fix live example at ":target" page

### PR DESCRIPTION
**Pull request by shapkarin**
Fix live example

ref issue #1985

### CODE REVIEW

**Comment by shapkarin** [comment]
closes #1985

File: files/ru/web/css/_colon_target/index.html [hunk 1]
```diff
@@ -29,7 +29,7 @@ <h2 id="Синтаксис">Синтаксис</h2>
 
 <h2 id="Примеры">Примеры</h2>
 
-<h3 id="Таблица_контента">Таблица контента</h3>
+<h3 id="a_table_of_contents">Таблица контента</h3>
```
Comment (sashasushko) [inline files/ru/web/css/_colon_target/index.html:32]:
```suggestion
<h3 id="A_table_of_contents">Таблица контента</h3>
```

File: files/ru/web/css/_colon_target/index.html [hunk 2]
```diff
@@ -71,7 +71,7 @@ <h4 id="CSS">CSS</h4>
 
 <h4 id="Результат">Результат</h4>
 
-<div>{{EmbedLiveSample('A_table_of_contents', 500, 300)}}</div>
+<div>{{EmbedLiveSample('a_table_of_contents', 500, 300)}}</div>
```
Comment (sashasushko) [inline files/ru/web/css/_colon_target/index.html:74]:
```suggestion
<div>{{EmbedLiveSample('A_table_of_contents', 500, 300)}}</div>
```

**Review by sashasushko — COMMENTED** [review]
I changed attribute name same in [en-us version](https://github.com/mdn/content/blob/main/files/en-us/web/css/_colon_target/index.html#L36)

**Review by sashasushko — APPROVED** [review]

**Comment by sashasushko** [comment]
@shapkarin спасибо.

**Pull request merged**